src/mobility/components/CarSharingStationBottomSheet.tsx

```diff
@@ -93,55 +93,79 @@ export const CarSharingStationSheet = ({stationId, distance, close}: Props) => {
                 <GenericSectionItem>
                   <OperatorLogo
                     operatorName={operatorName}
+                    maxHeight={20}
+                    maxWidth={20}
                     logoUrl={brandLogoUrl}
                   />
+                  <View style={style.stationText}>
+                    <ThemeText type="body__secondary" color="secondary">
+                      {stationName}
+                    </ThemeText>
+                    <WalkingDistance distance={distance} />
+                  </View>
                 </GenericSectionItem>
-              </Section>
-              {isAnyAvailable(station.vehicleTypesAvailable) &&
-                station.vehicleTypesAvailable
-                  ?.filter(isAvailable)
-                  .sort(byName(language))
-                  .map((vehicle, i) => (
-                    <Section key={'vehicle' + i} style={style.carSection}>
-                      <GenericSectionItem>
-                        <View style={style.carDetailsContainer}>
-                          <View style={style.carImage}>
+                <GenericSectionItem>
+                  <View style={style.carSection}>
+                    <View>
+                      <View style={style.availableCarSection}>
+                        <ThemeIcon
+                          style={style.icon}
+                          svg={Car}
+                          fill={theme.text.colors.secondary}
+                        />
+                        <ThemeText
+                          type="body__secondary--bold"
+                          color="secondary"
+                        >
+                          {t(
+                            CarSharingTexts.stations.carsAvailable(
+                              totalAvailableCars(station.vehicleTypesAvailable),
+                              station.capacity,
+                            ),
+                          )}
+                        </ThemeText>
+                      </View>
+                      <ThemeText type="body__secondary" color="secondary">
+                        {t(CarSharingTexts.stations.carsAvailableLabel)}
+                      </ThemeText>
+                    </View>
+                    <View style={style.carDetailsContainer}>
+                      {station.vehicleTypesAvailable
+                        ?.slice(station.capacity > 2 ? -1 : -2)
```

Comment:
I don't actually get this part. Maybe move it out into a utility function, where you can add jsdoc regarding what is actually happening 😅

Reply:
refactored and added documentation